Build/Projects/FrameworkReferences.definition

```diff
@@ -15,6 +15,16 @@
   <Platform Type="Linux">
     <Reference Include="System" />
     <Reference Include="System.Runtime.Serialization" />
+
+    <Service Name="MonoGame.Framework/OpenALAudio">
+      <NativeBinary Path="ThirdParty/Dependencies/openal-soft/Linux/x86/libopenal.so.1" Anchor="x86" />
+      <NativeBinary Path="ThirdParty/Dependencies/openal-soft/Linux/x64/libopenal.so.1" Anchor="x64" />
+    </Service>
+
+    <Service Name="MonoGame.Framework/SDLWindowing">
```

Comment:
Redundant inclusion? since bellow you specify this stuff in `<Service Name="MonoGame.Framework/DesktopGL">`.

Also no point in defining service for platform references since it only has 1 option and is always gonna be true.

Reply:
Oh, right. I forgot to remove this.